frontend/src/app/main/ui/components/numeric_input.cljs

```diff
@@ -53,14 +53,13 @@
 
         ;; This `value` represents the previous value and is used as
         ;; initil value for the simple math expression evaluation.
-        value       (d/parse-double value-str default)
+        value       (if (not= :multiple value-str) (d/parse-double value-str default) nil)
```

Comment:
if you have a `(if <condition> <body> nil)` you can use: `(when <condition> <body>)` is the same thing because `when` returns `nil` if the condition is not fulfilled